src/documents/signals/handlers.py

```diff
@@ -570,6 +574,32 @@ def run_workflows_updated(sender, document: Document, logging_group=None, **kwar
     )
 
 
+@shared_task(
```

Comment:
This is pretty neat actually.  I didn't know about it.  I think we could use some more of the automatic handling, while still logging.

If we catch the error, log, then raise again, we could use `autoretry_for`, `max_retries` and `retry_backoff`

Reply:
Yea they're cool. That's actually initially how I had it but I wasn't so sure about the logging / error handling. Take a look and let me know if this looks right to you.

Reply:
I think we want just `max_retries`, not the `retry_kwargs`.  Not 100%, but I believe that applies only if the task calls `.retry`

Reply:
Oh yea, I think so thanks